mne/decoding/tests/test_time_gen.py

```diff
@@ -70,7 +70,15 @@ def test_generalization_across_time():
     assert_equal("<GAT | fitted, start : -0.200 (s), stop : 0.499 (s), no "
                  "prediction, no score>", '%s' % gat)
     assert_equal(gat.ch_names, epochs.ch_names)
+    # test different predict function:
+    gat.decision_function(epochs)
+    assert_array_equal(np.shape(gat.y_pred_), (15, 15, 14, 1))
+    gat.predict_proba(epochs)
+    assert_array_equal(np.shape(gat.y_pred_), (15, 15, 14, 2))
```

Comment:
My preference would be storing it in the gat object because that way the API across scoring probabilities and predictions would be consistent, because a new and external function IMO doesn't fit with MNE style as well as a method on gat, and also because I'm lazy :).
I mean, gat objects aren't perfectly equivalent to scikit-learn models to begin with.

But either way is fundamentally fine.